src/cache_cf.cc

```diff
@@ -3072,7 +3072,7 @@ free_time_msec(time_msec_t * var)
 static void
 dump_time_nanoseconds(StoreEntry *entry, const char *name, const std::chrono::nanoseconds &var)
 {
-    storeAppendPrintf(entry, "%s %" PRId64 " nanoseconds\n", name, var.count());
+    storeAppendPrintf(entry, "%s %" PRId64 " nanoseconds\n", name, static_cast<int64_t>(var.count()));
```

Comment:
A slightly better fix may be:

```suggestion
    // This assumes that nothing in C++ is larger than long long. TODO: Avoid printf().
    storeAppendPrintf(entry, "%s %lld nanoseconds\n", name, static_cast<long long>(var.count()));
```

The above may be slightly better because there is nothing larger than `long long` in today's C++. The current PR code might, in theory, truncate the count() value on rare platforms where `long long` is 128 bits rather than 64. All this is rather theoretical, of course, so I do not insist on this change.

Reply:
according to the freebsd documentation, lld should work. I'm accepting your suggestion, althought I agree the scenario is very theoretical

Reply:
We used to support some pretty bizarre architectures, but by now there's just a few that are relevant.
I used the "Commit suggestion" button in github. No idea were that line came from

Reply:
> I used the "Commit suggestion" button in github. No idea were that line came from

Must be a GitHub bug then. I [re-added](https://github.com/squid-cache/squid/pull/577/commits/82c70e9b2996ba749a1ef34e55279c23f42efc8a) the line that GitHub removed.

AFAICT, the ball is fully on @yadij side now.